lib/sentry/event.ex

```diff
@@ -180,4 +181,7 @@ defmodule Sentry.Event do
   @spec culprit_from_stacktrace(Exception.stacktrace) :: String.t | nil
   def culprit_from_stacktrace([]), do: nil
   def culprit_from_stacktrace([{m, f, a, _} | _]), do: Exception.format_mfa(m, f, a)
+
+  defp arity_to_integer(arity) when is_list(arity), do: Enum.count(arity)
+  defp arity_to_integer(arity), do: arity
```

Comment:
I had that but decided against it since we weren't checking for it before and so I wasn't sure how safe it would be.  Erlang's typespecs say it should be an integer 0..255 or a list, so we could, but I erred on the side of caution.

I will switch it back though.